packages/account-web-vault/src/web-user.ts

```diff
@@ -0,0 +1,250 @@
+import { Network, Context, Profile } from '@verida/client-ts';
+import { VaultAccount, hasSession, VaultAccountConfig } from './index';
+import { EventEmitter } from 'events'
+import { MessageSendConfig } from '@verida/client-ts/dist/context/interfaces';
+
+const VeridaEvents = new EventEmitter()
+
+export interface WebUserProfile {
+    name?: string
+    avatarUri: string
+    country?: string
+    description?: string
+}
+
+export interface WebUserConfig {
+    accountConfig: VaultAccountConfig
+    clientConfig: ClientConfig
+    contextConfig: ContextConfig
+    networkConfig: NetworkConnectionConfig
+}
+
+export interface WebUserMessage {
+    subject: string
+    text: string
+    link?: WebUserMessageLink
+}
+
+export interface WebUserMessageLink {
+    url: string
+    text: string
+}
+
+/**
+ * 
+ * @event profileChanged
+ * @event connected
+ * @event disconnected
+ */
+export class WebUser {
```

Comment:
Just adding, it may have some impacts on how to handle such static class with a global/local state like React State or React Context.

Reply:
Yes, exactly the rationale.

I’m flexible with either. 

You’re suggesting a singleton because it will play nicer with React?

I originally wrote this within a React app.

Reply:
As discussed, will stick with static as is.